dataland-backend/src/main/resources/application.properties

```diff
@@ -11,7 +11,7 @@ server.servlet.context-path=/api
 dataland-edc-client.baseUrl=http://dataland-edc:9191/api
 spring.security.oauth2.resourceserver.jwt.issuer-uri=https://${PROXY_PRIMARY_URL:local-dev.dataland.com}/keycloak/realms/datalandsecurity
 spring.security.oauth2.resourceserver.jwt.jwk-set-uri=http://keycloak:8080/keycloak/realms/datalandsecurity/protocol/openid-connect/certs
-dataland.authorization.publiclinks=/actuator/health,/actuator/health/ping,/actuator/info,/swagger-ui/**,/v3/api-docs/**,/companies/**,/data/**,/metadata,/metadata/**
+dataland.authorization.publiclinks=/actuator/health,/actuator/health/ping,/actuator/info,/swagger-ui/**,/v3/api-docs/**
 dataland.authorization.internallinks=/internal/**
```

Comment:
Why do we keep a seperate list `internallinks` besides `publiclinks`, when we concatenate the values of both lists in the only code occurence of `internallinks`:

![image](https://github.com/d-fine/Dataland/assets/99255296/4e79308c-a8b0-4baf-8d06-0ead0422029a)

Reply:
I am also wondering why the `internal/cached/{dataId}` endpoint is public anyways.
Won't it be only accessed by the internal-storage service?

Reply:
Which mechanisms have been repsonsibe to avoid strangers from the outside to pick up cached data via dataId until now?

Reply:
Phone call needed to accept this.

Reply:
Ergebnis:
- Der "/internal" Endpunkt wird durch nginx nach außen geschützt
- Er ist ansonsten "public", damit die anderen Services ohne bearer-token o.Ä. dran kommen

=> D.h. es stellt sich eigentlich nur die Frage, ob wir die zwei unterschiedlichen Listen hier zu Leserlichkeits-Zwecken so behalten wollen.  Rein technisch werden sie sowieso concatenated und somit landen alle ihre Elemente im selben Pool.

Das könnt ihr entscheiden, weil ich da keine starke Meinung zu habe.

Reply:
Bevor das geändert wird muss aber sichergestellt werden, dass das Aufplitten der swagger Ui in zwei Gruppen, von denen nur eine sichtbar ist, noch funktioniert. Wir hatten diese ganze Linkgeschichte nämlich im Rahmen, dass die internen Endpunkte nicht in der SwaggerUi auftauchen sollen eingeführt. Ob die Unterscheidung hier einen Einfluss darauf hat, weiß ich nicht, aber der Punkt mit der SwaggerUi sollte bedacht werden bevor das hier geändert wird.

Reply:
Ok 👍